napari/_qt/dialogs/qt_plugin_dialog.py

```diff
@@ -353,14 +434,31 @@ def addItem(
             plugin_name=plugin_name,
             enabled=enabled,
         )
-        method = getattr(
-            self.installer, 'uninstall' if plugin_name else 'install'
+        item.widget = widg
+        # method = getattr(
+        #     self.installer, 'uninstall' if plugin_name else 'install'
+        # )
```

Comment:
As above — remove comments or uncomment, not sure here. =)

Reply:
Fixed!